onnxruntime/contrib_ops/cpu/reverse.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/framework/op_kernel.h"
+#include <unordered_set>
+#include "core/util/eigen_common_wrapper.h"
+
+namespace onnxruntime {
+namespace contrib {
+
+// some templated aliases
+template <typename T, int rank>
```

Comment:
Do any of these things apart from the Reverse class declaration need to be in the .h file, or can they move to the .cc?

Reply:
Moved.